src/components/Button.tsx

```diff
@@ -1,10 +1,13 @@
 interface ButtonProps {
-    setFunction: () => void;
-    text: string;
+  setFunction: (arg: number) => void;
+  text: string;
 }
 
 const Button = ({ setFunction, text }: ButtonProps) => {
```

Comment:
```suggestion
import { ButtonHTMLAttributes } from "react";

type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement>

export const Button = (props: ButtonProps) => {
  const { children, ...rest } = props;
  return <button {...rest}>{children}</button>;
};
```

Почитай про зарезервированный пропс children:
https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children
https://ru.legacy.reactjs.org/docs/composition-vs-inheritance.html

Reply:
fixed